README.md

```diff
@@ -6,10 +6,11 @@
 [//]: # "  "
 [//]: # " SPDX-License-Identifier: BSD-3-Clause "
 
-[![Build Status](https://travis-ci.com/eclipse-ee4j/jaxb-api.svg?branch=master)](https://travis-ci.com/eclipse-ee4j/jaxb-api)
-
 # Jakarta XML Binding project
 
+[![Build Status](https://travis-ci.com/eclipse-ee4j/jaxb-api.svg?branch=master)](https://github.com/eclipse-ee4j/jaxb-api/actions)
```

Comment:
img url is: https://github.com/eclipse-ee4j/jaxb-api/actions/workflows/maven.yml/badge.svg?branch=masterstatus url is: https://github.com/eclipse-ee4j/jaxb-api/actions/workflows/maven.yml?branch=master